contribute/style-guides/themes.md

```diff
@@ -98,29 +104,38 @@ BarStories.add('Story' () => {
 
 ### Using themes in Angular code
 
-There should be very few cases where theme would be used in Angular context. For this purpose there is a function available that retrieves current theme: `import { getCurrentTheme } from app/core/utils/ConfigProvider`. Angular components should be migrated to React, or if that's not possible at the moment, styled using SASS.
+There should be very few cases where a theme would be used in an Angular context. For this purpose, there is a function available that retrieves the current theme:
 
+```ts
+import { getCurrentTheme } from app/core/utils/ConfigProvider
+```
+
+Angular components should be migrated to React, or if that's not possible at the moment, styled using Sass.
 
 ## FAQ
-### How can I modify SASS variable files?
+
+### How can I modify Sass variable files?
+
 > For the following to apply you need to run `yarn dev` task.
 
-`[_variables|_variables.dark|_variables.light].generated.scss` files are the ones that are referenced in the main SASS files for SASS variables to be available. **These files are automatically generated and should never be modified by hand!**.
+`[_variables|_variables.dark|_variables.light].generated.scss` files are the ones that are referenced in the main Sass files for Sass variables to be available. **These files are automatically generated and should never be modified by hand!**
```

Comment:
SASS is an acronym. All caps throughout.

Reply:
I believe it's a backronym... Either way, the language name is currently capitalized like "Sass": https://github.com/grafana/grafana/pull/22381/files/a44bb6cd4f96ed7eb318b8145f7d08004e049289#r383376872

Reply:
The name of the [project](https://sass-lang.com/) is indeed Sass, so I agree with @robatron - I don't see any reason to diverge.